challenges/qr_code_component/index.html

```diff
@@ -27,36 +24,32 @@
     />
     <link rel="manifest" href="../site.webmanifest" />
     <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Outfit:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body class="m-0">
-    <div
-      class="flex h-[100vh] w-full flex-col place-content-center text-center"
-    >
-      <div>
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://www.frontendmentor.io/challenges/qr-code-component-iux_sIO_H"
-          target="_blank"
-          >Frontend Mentor challenge</a
-        >
-      </div>
-      <div>
-        for
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://github.com/optimumBA"
-          target="_blank"
-          >Optimum BH</a
-        >
-        Internship.
-      </div>
+  <body class="bg-light-gray">
+    <div class="mx-7 flex h-screen items-center justify-center">
+      <div class="max-w-xs rounded-2xl bg-white p-4 drop-shadow">
+        <img
+          class="rounded-xl"
+          src="../images/image-qr-code.png"
+          alt="QR code to scan"
+        />
+
+        <div class="my-3 p-3 text-center">
+          <h1 class="font-outfit text-xl font-semibold">
+            Improve your front-end skills by building projects
+          </h1>
 
-      <div class="mt-3">
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://github.com/optimumBA/frontend_mentor_challenges/tree/qr_code_component/challenges/qr_code_component"
-          >Source code</a
-        >
+          <p class="font-outfit text-grayish-blue mt-3">
+            Scan the QR code to visit Frontend Mentor and take your coding
+            skills to the next level
+          </p>
+        </div>
```

Comment:
There should also be credits visible on the page with the link to the source code in your repo for the tests to pass.